proxy/logging/LogField.cc

```diff
@@ -772,7 +772,9 @@ LogFieldList::marshal_len(LogAccess *lad)
   int bytes = 0;
   for (LogField *f = first(); f; f = next(f)) {
     if (f->type() != LogField::sINT) {
-      bytes += f->marshal_len(lad);
+      const int len = f->marshal_len(lad);
```

Comment:
This is not the only logging anomaly we are seeing so I suspect it is in fact an issue elsewhere. I would rather do a basic fix rather than patching after the fact.